### PR TITLE
🔨 Update references to HMMPS Auth, it's a publicly accessible URL so mo…

### PR DESCRIPTION
**Pull request by nickmcmahon01**
…ving it out of secrets, security by obscurity isn't security.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -37,12 +43,12 @@ module.exports = {
...
     },
     apiClientId: get('API_CLIENT_ID', 'my-diary', requiredInProduction),
-    apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+    apiClientSecret: get('API_CLIENT_SECRET', 'kJwGCYAhn4AYRQF9', requiredInProduction),
   },
   app: {
     production,
     mailTo: process.env.MAIL_TO || 'feedback@digital.justice.gov.uk',
-    url: process.env.CHECK_MY_DIARY_URL || `http://localhost:${process.env.PORT || 3005}`,
+    url: process.env.CHECK_MY_DIARY_URL || 'http://localhost:3000',
```
Comment (astroash) [inline config.js:51]: This is a bit messy, but it's defaulting to the testing port if no env `PORT` is set

File: config.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const requiredInProduction = { requireInProduction: true }
 const noFallbackInProduction = { noFallbackInProduction: true }
 
 module.exports = {
-  sessionSecret: get('SESSION_SECRET', 'app-insecure-default-session', requiredInProduction),
+  sessionSecret: get('SESSION_SECRET', '456456453rtretretete', requiredInProduction),
```
Comment (astroash) [inline config.js:19]: Is this not an actual secret?

File: config.js [hunk 3]
```diff
@@ -37,12 +43,12 @@ module.exports = {
       freeSocketTimeout: 30000,
     },
     apiClientId: get('API_CLIENT_ID', 'my-diary', requiredInProduction),
-    apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+    apiClientSecret: get('API_CLIENT_SECRET', 'kJwGCYAhn4AYRQF9', requiredInProduction),
```
Comment (astroash) [inline config.js:46]: Is this not an actual secret?

File: config.js [hunk 4]
```diff
@@ -56,19 +62,19 @@ module.exports = {
     healthCheckUrl: process.env.NOTIFY_HEALTH_CHECK_URL || 'https://api.notifications.service.gov.uk/_status',
   },
   cmdApi: {
-    url: get('CMD_API_URL', 'http://localhost:9191'),
+    url: get('CMD_API_URL', 'http://localhost:8081'),
```
Comment (astroash) [inline config.js:65]: I _think_ this port is needed for integration tests, but don't quote me on that one

File: config.js [hunk 5]
```diff
@@ -56,19 +62,19 @@ module.exports = {
...
   },
   hmppsCookie: {
-    name: process.env.HMPPS_COOKIE_NAME || 'hmpps-session-dev',
-    domain: process.env.HMPPS_COOKIE_DOMAIN || 'localhost',
+    name: process.env.HMPPS_COOKIE_NAME || 'check-my-diary-dev',
+    domain: process.env.HMPPS_COOKIE_DOMAIN || 'http://localhost:3000',
     expiryMinutes: process.env.WEB_SESSION_TIMEOUT_IN_MINUTES || 20,
   },
-  port: get('PORT', 3005, requiredInProduction),
+  port: get('PORT', 3000, requiredInProduction),
```
Comment (astroash) [inline config.js:72]: Default here needs to be `3005` for integration tests

File: helm_deploy/check-my-diary/templates/_envs.tpl [hunk 6]
```diff
@@ -17,10 +17,7 @@ env:
         key: API_CLIENT_SECRET
 
   - name: API_AUTH_ENDPOINT_URL
-    valueFrom:
-      secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: API_AUTH_ENDPOINT_URL
+    value: {{ .Values.env.TWO_FACT_AUTH_ON | quote }}
```
Comment (jonhindle) [inline helm_deploy/check-my-diary/templates/_envs.tpl:20]: should this be API_AUTH_ENDPOINT_URL

**Pull request merged**